docs/synapse/userguides/userguide_section7.rst

```diff
@@ -0,0 +1,183 @@
+  
+- **Organizations.** It is difficult to determine a unique characteristic for an organization (business, non-profit, government department, NGO). The organization name is one possibility, but names are not universally deconflicted – it is possible for more than one Acme Corporation to exist globally. Some type of business identification number is another possibility – a tax ID number, or a Dun & Bradstreet number, for example – but those may be country-specific and / or entity-specific (applying to businesses but not to government departments, for example). For these reasons, the primary property of an organization in the Synapse data model is represented by a randomly-generated GUID (``ou:org=<guid>``). Secondary properties may include an organization's name, its location / country of origin, or the URL of the organization's web site.
+
+- **People.** Similar to an organization, it is difficult to determine a universally unique characteristic for a person. Names are not guaranteed to be unique, as there can easily be more than one John Smith in the world. The same problem occurs with date of birth (multiple people can share the same birth date), and even the combination of name and date of birth cannot be guaranteed to be unique. For this reason, a person is also represented by a randomly-generated GUID (``ps:person=<guid>``). Secondary properties may include the person's given, middle, and family names, date of birth, and so on.
+
+**Aliases**
+
+GUIDs are used as primary properties for forms which are not easily deconflictable (that is, which do not have readily identified characteristics that can serve as a unique primary property). For commonly used node types that are represented by GUIDs (such as organizations or people), it is impractical for an analyst to have to type (or even copy / paste) a GUID at the CLI every time they want to reference a given node. While nodes can be retrieved based on a more "human-friendly" secondary ``<prop>=<value>`` (e.g., ``ou:org:alias=vertex``), even the need to enter a full secondary property / value may be inconvenient.
+
+Synapse allows an **alias** to be defined as part of the type for the form's primary property. The alias is typically one of the form's secondary properties. For example, the Synapse data model defines the property ``ps:person:guidname`` as an alias for a person, and ``ou:org:alias`` as an alias for an organization. An example of the alias definition can be seen in this snippet of source code that defines the ``ou:org`` type (from ``orgs.py`` - emphasis added):
+::
+  def getBaseModels():
+    modl = {
+      'types': (
+        ('ou:org', {'subof': 'guid', **'alias': 'ou:org:alias',**
+              'doc': 'A GUID for a human organization such
+              as a company or military unit'
```

Comment:
I think this has to be moved up onto the next line for formatting to display correctly.